src/main/resources/application.yml

```diff
@@ -83,6 +83,9 @@ management:
       cache:
         time-to-live: 2000ms
 
+springdoc:
+  remove-broken-reference-definitions: false
```

Comment:
The magic words that allow for apparently orphaned types to be included in the Schema.